internal/pkg/gateway/commit/finder_test.go

```diff
@@ -23,6 +23,21 @@ type queryProvider interface { // Mimic QueryProvider to avoid circular import w
 }
 
 func TestFinder(t *testing.T) {
+	sendUntilDone := func(commitSend chan<- *ledger.CommitNotification, msg *ledger.CommitNotification) chan struct{} {
+		done := make(chan struct{})
+
+		go func() {
+			for {
+				select {
+				case commitSend <- msg:
```

Comment:
This will result in a tight loop, no?  Probably worth putting at least a small sleep in here? 10ms?

Reply:
👍